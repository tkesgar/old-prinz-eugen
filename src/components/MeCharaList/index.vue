<template>
  <me-chara-list />
</template>

<script>
import MeCharaList from './view'

export default {
  components: {
    MeCharaList
  }
}
</script>
