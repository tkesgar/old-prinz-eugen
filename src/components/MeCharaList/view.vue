<template>
  <div class="chara-list-empty text-center">
    <p>Anda tidak memiliki karakter.</p>
    <p>Silakan buat karakter baru <b-link to="/chara/new">di sini</b-link>.</p>
  </div>
</template>

<script>
export default {
  props: {
    charas: {
      type: Array,
      default: () => ([])
    }
  }
}
</script>
