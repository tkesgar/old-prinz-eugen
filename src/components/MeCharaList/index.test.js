import { shallowMount } from '@vue/test-utils'
import MeCharaList from './'

describe('MeCharaList', () => {
  const wrapper = shallowMount(MeCharaList)

  test('should mount successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
