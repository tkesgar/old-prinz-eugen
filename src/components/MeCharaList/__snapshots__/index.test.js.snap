// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeCharaList should match snapshot 1`] = `
<me-chara-list-stub
  charas=""
/>
`;
